src/utils/pretty-date.js

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+function prettyDate (date, locales) {
```

Comment:
Super nit, but (especially without TypeScript) I find it a helpful convention to start function names with a verb, to make them jump out when they're not called - so this could e.g. be `formatDate` or similar.